src/main/java/mate/academy/service/MovieService.java

```diff
@@ -9,4 +9,8 @@ public interface MovieService {
     Movie get(Long id);
 
     List<Movie> getAll();
+
+    Boolean update(Movie movie);
```

Comment:
Consider changing the return type from `Boolean` to `boolean` for the `update` method to avoid unnecessary boxing.